Do not special-case macOS to use a static library; use a shared library
like everybody else does.
--- modules/FindLibUSB.cmake.orig	2018-12-12 04:55:19.000000000 -0600
+++ modules/FindLibUSB.cmake	2020-03-24 01:51:21.000000000 -0500
@@ -74,9 +74,7 @@
 	)
 endif()
 
-if (APPLE)
-	set(LIBUSB_NAME libusb-1.0.a)
-elseif(MSYS OR MINGW)
+if(MSYS OR MINGW)
 	set(LIBUSB_NAME usb-1.0)
 elseif(WIN32 OR CMAKE_VS_PLATFORM_NAME)
 	set(LIBUSB_NAME libusb-1.0.lib)
